README.md

```diff
@@ -138,7 +138,7 @@ The default webpack config shipped with scriptoni should be fine in most cases.
 If this is the case, you can override the default config by passing an additional `--webpackConfig` argument, followed by the file path containing your override function.
 
 Let's say, for example, you want to change the output library.
-You can provide a `webpack.config.js` file in the root directory of your project, with the following content:
+You can provide a `webpack.dev.js` file in the root directory of your project, with the following content:
```

Comment:
why this change? It looks like you only renamed the internal "webpack config for dev" file, while you didn't change the logic to retrieve the user-provided file in any way. This should stay `webpack.config.js` (also because there is a single file both for dev and for build!)